_tour/operators.md

```diff
@@ -67,12 +115,27 @@ When an expression uses multiple operators, the operators are evaluated based on
 |
 (all letters, $, _)
```

Comment:
Could you also double check https://github.com/scala/docs.scala-lang/issues/2509#issue-1359220737?

Reply:
Ok. I will change the doc according to the specification.